node/lib/task.ts

```diff
@@ -591,6 +591,47 @@ export function exist(path: string): boolean {
 }
 
 /**
+ * Reads a directory's contents.
+ * see [fs.readdirsync](https://nodejs.org/api/fs.html#fs_fs_readdirsync_path_options)
+ * 
+ * @param       path    path of directory to readdirSync
+ * @returns     array of directory contents
+ */
+export function readdirSync(path: string): string[] {
```

Comment:
we have been wrapping shelljs with other disk operations.  That would mean we add ls function, not readdirSync.